nvme-rpmb.c

```diff
@@ -292,25 +292,10 @@ static int send_rpmb_req(nvme_link_t l, unsigned char tgt, int size,
 	return nvme_security_send(l, &args);
 }
 
-static int recv_rpmb_rsp(nvme_link_t l, int tgt, int size,
-			 struct rpmb_data_frame_t *rsp)
+static int recv_rpmb_rsp(nvme_link_t l, int tgt, int size, struct rpmb_data_frame_t *rsp)
 {
-
-	struct nvme_security_receive_args args = {
-		.args_size	= sizeof(args),
-		.nsid		= 0,
-		.nssf		= tgt,
-		.spsp0		= RPMB_NVME_SPSP,
-		.spsp1		= 0,
-		.secp		= RPMB_NVME_SECP,
-		.al		= 0,
-		.data_len	= size,
-		.data		= (void *)rsp,
-		.timeout	= NVME_DEFAULT_IOCTL_TIMEOUT,
-		.result		= NULL,
-	};
-
-	return nvme_security_receive(l, &args);
+	return nvme_security_receive(l, 0, 0, tgt, RPMB_NVME_SPSP, 0, RPMB_NVME_SECP, rsp, size,
```

Comment:
The argument order got mangled up:
``` suggestion
	return nvme_security_receive(l, 0, tgt, RPMB_NVME_SPSP, 0, RPMB_NVME_SECP, 0, (void *)rsp, size,
```
spsp0 and spsp1 will be merged into one as per https://github.com/nvme-experiments/libnvme/pull/31